projects/storefrontstyles/scss/components/product/list/_active-facets.scss

```diff
@@ -14,19 +14,16 @@
       display: none;
     }
   }
-
```

Comment:
Why change this file? This doesn't seem to fit the scope of the PR.
Let's discuss.

Reply:
Don't know why this is here. I didn't make any changes to this file. Maybe during a /develop branch merge it changed these files

Reply:
In an earlier commit when merging a branch to this one, these changes existed. The revert of that commit is undoing those changes